<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brusilovskiy</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="temperature_label">
        <property name="text">
         <string>Введите значение давления в К</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="components_file_label">
        <property name="text">
         <string>Выберите файл с составом и параметрами смеси</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="components_file_box"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="components_file_but">
        <property name="text">
         <string>Обзор</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="binary_file_but">
        <property name="text">
         <string>Обзор</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="binary_file_label">
        <property name="text">
         <string>Выберите файл с коэйффициентами бинарного взаимодействия</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="pressure_label">
        <property name="text">
         <string>Введите значение давления в МПа</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="binary_file_box"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="pressure_box"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="temperature_box"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="calculate_button">
      <property name="text">
       <string>Произвести расчёт</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
